source/api/presentation/3.0/index.md

```diff
@@ -47,38 +47,38 @@ This is a work in progress and may change without any notices. Implementers shou
 
-The following are within the scope of the current document:
+Implementations of this specification will be able to:
 
-  * The display of digitized images associated with a particular physical object, or born-digital compound object.
-  * Navigation between multiple views of the object.
-  * The display of text, and resources of other media types, associated with the object views – this includes descriptive information about the object, labels that can aid navigation such as numbers associated with individual pages, copyright or attribution information, etc.
+  * display to the user digitized images, video, audio and other content types associated with a particular physical object, or born-digital compound object ; 
+  * allow the user to navigate between multiple views of the object, either sequentially or hierarchically ;
+  * and display descriptive information about the object, view or navigation structure to provide context to the user.
 
 The following are __not__ within scope:
 
-  * The discovery or selection of interesting digitized objects is not directly supported; however hooks to reference further resources are available.
+  * The discovery or selection of interesting objects is not directly supported; however hooks to reference further resources are available.
   * Search within the object; which is described by the [IIIF Content Search API][search-api].
 
-Note that in the following descriptions, "object" (or "physical object") is used to refer to a physical object that has been digitized or a born-digital compound object, and "resources" refer to the digital resources that are the result of that digitization or digital creation process.
+Note that in the following descriptions, "object" is used to refer to the object that has been digitized or a born-digital compound object, and "resources" refer to the digital resources and content that are the result of that digitization or digital creation process.
 
 ###  1.2. Motivating Use Cases
 
-There are many different types of digitized or digital compound objects, from ancient scrolls to modern newspapers, from medieval manuscripts to online comics, and from large maps to small photographs. Many of them bear texts, sometimes difficult to read either due to the decay of the physical object or lack of understanding of the script or language.  These use cases are described in a separate [document][prezi-use-case-doc].
+There are many different types of digitized or digital compound objects, from ancient scrolls to modern newspapers, from operas to pop music, and from the earliest black and white film to to computer generated animations. Many of them bear text, sometimes difficult to read either due to the decay of the physical object or lack of understanding of the script or language.  These use cases are described in a separate [document][prezi-use-case-doc].
```

Comment:
(I think we can work at these examples more, to cover the outer product of [Galleries, Libraries, Archives, Museums] and [Images, Audio, Video] with a few ingenious choices)

A suggestion (not necessarily a requested change):

There are many different types of digitized or digital compound objects; ancient scrolls, paintings, letters, books, newspapers, films, operas, albums, field recordings and computer generated animations. Many of them bear the written or spoken word, sometimes difficult to read or hear either due to the decay of the physical object or lack of understanding of the script or language.

Reply:
:+1: